openapi/endpoints/funding-sources/models/funding-source.yaml

```diff
@@ -0,0 +1,31 @@
+type: object
+properties:
+  id:
+    type: string
+    description: Funding source primary identifier
+    example: 91ad6fea3b52ca58d60d7fd310f789ec
+  accountId:
+    type: string
+    description: Cardholder account this funding source belongs to
+    example: 057aa15913a57f50577234c8426534c0
+  externalId:
+    type: string
+    description: The identifier for this funding source. It would be the wallet address making a deposit on the ImmersvePaymentProtocol
+    example: '0xbDA5747bFD65F08deb54cb465eB87D40e51B197E'
```

Comment:
```suggestion
```
This field is no longer on the api response. Slack [convo](https://immersve.slack.com/archives/C05K7NANW0L/p1692331582312009?thread_ts=1692320759.240369&cid=C05K7NANW0L)

Reply:
It was reintroduced with create funding source endpoint https://github.com/immersve/amethyst/blob/main/src/models/FundingSource.ts#L161
There was a conversation (couldn't find it in writing) about having the response symmetrical to the input parameters when creating entities, and this field was added back.

Reply:
The create funding source endpoint doesn't have an externalId in the payload. So how is it symmetrical?

Reply:
Why do you think it doesn't? It's using `toApiResponse` when returning newly created funding source. If you check endpoint response in cucumber tests it will have externalId.
![Screenshot 2023-08-29 at 9 04 58 AM](https://github.com/immersve/immersve-docs/assets/4097083/42f47fe1-e7d8-496a-b375-825be99e27bc)